test/integration/Templates.test.ts

```diff
@@ -90,4 +92,25 @@ describe('Client - Templates', function () {
             expect(result.Message).to.eq('OK');
         });
     });
+
+    describe('push templates', () => {
+        it("non existing servers error", () => {
+            let pushRequest = new TemplatesPushRequest(0,1, false);
```

Comment:
No space between `0,1`

Reply:
Also `const` instead of `let` would be better.

Reply:
fixed by tslint